src/wp-includes/deprecated.php

```diff
@@ -4225,3 +4225,63 @@ function wp_render_duotone_filter_preset( $preset ) {
 	_deprecated_function( __FUNCTION__, '5.9.1', 'wp_get_duotone_filter_property()' );
 	return wp_get_duotone_filter_property( $preset );
 }
+
+/**
+ * Checks whether serialization of the current block's border properties should
+ * occur.
+ *
+ * @since 5.8.0
+ * @access private
+ * @deprecated 6.0.0 Use `wp_should_skip_block_supports_serialization` introduced in 6.0.0.
+ *
+ * @param WP_Block_Type $block_type Block type.
+ * @return bool Whether serialization of the current block's border properties
+ *              should occur.
+ */
+function wp_skip_border_serialization( $block_type ) {
+	_deprecated_function( __FUNCTION__, '6.0.0', 'wp_should_skip_block_supports_serialization()' );
+	$border_support = _wp_array_get( $block_type->supports, array( '__experimentalBorder' ), false );
+
+	return is_array( $border_support ) &&
+		array_key_exists( '__experimentalSkipSerialization', $border_support ) &&
+		$border_support['__experimentalSkipSerialization'];
```

Comment:
Nitpick: is there a reason not to use `empty`?
```suggestion
	return is_array( $border_support ) && ! empty ( $border_support['__experimentalSkipSerialization'] );
```

Reply:
I don't think there's a reason not to use `empty()`, but given the only change was to deprecate this method, then I wouldn't have thought that we need to worry about refactoring it. 🤷

Reply:
Sure. This was more of a nitpick than a serious request to change this.
I'm just not seeing the need for calling `array_key_exists` here. But it should work absolutely fine in its current state.